include/Mixer.h

```diff
@@ -315,6 +355,17 @@ class LMMS_EXPORT Mixer : public QObject
 	void requestChangeInModel();
 	void doneChangeInModel();
 
+	RequestChangesGuard requestChangesGuard()
+	{
+		return RequestChangesGuard{this};
+	}
+
+	template<class Callable, class ReturnT=decltype(std::declval<Callable>()())>
+	ReturnT runWhileNotRendering(Callable &&callable) {
```

Comment:
```suggestion
	ReturnT runWhileNotRendering(Callable &&callable)
	{
```
I'm also a curly-brackets-on-the-same-line type of person, but almost all the source files of LMMS puts them in new line. I would recommend doing so for the sake of consistency.